stations.csv

```diff
@@ -70515,3 +70515,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74661;Firenze Piazzale Montelungo;firenze-piazzale-montelungo;;;43.780650;11.246700;8433;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;@ITFLRPMO;t;24648;t;;f;;f;;f;;f;8314717;t;;;;;f;;f;;f;;f;;f;;f;f;;Florenz;Florence;Florencia;Florence;;;Florence;Florencie;;;フィレンツェ;피렌체;Firence;Florença;Флоренция;Florens;Floransa;佛罗伦萨;urn:trainline:public:nloc:csv74661;
 74662;Senigallia Lungomare Sud;senigallia-lungomare-sud;;;43.703090;13.243700;20267;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8314768;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;セニガッリア;;;;Сенигаллия;;;;urn:trainline:public:nloc:csv74662;
 74663;Palermo De Gasperi;palermo-de-gasperi;;;38.1480195;13.3285637;22186;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8312065;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;Palerme;;;;;;;パレルモ;팔레르모;;;Палермо;;;巴勒莫市;urn:trainline:public:nloc:csv74663;
+74664;Casamarciano;casamarciano;;;40.93125823842119;14.548592083542841;;IT;Europe/Paris;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74664;
```

Comment:
:thinking: Where are the coordinates coming from? Can you pls trim them down to 6 digits after the `.` max?

Reply:
We should have a test for that :sparkles: :smiling_imp:

Reply:
Thank you! Trimmed it to 6 digits after the `.`